src/app.js

```diff
@@ -73,7 +73,8 @@ export const start = args => {
       branchPayload = data[0];
 
       remoteList = data[1];
-
+      screen.append(branchTable);
+      screen.append(statusBarContainer);
```

Comment:
@JayaKrishnaNamburu 	Can we actually move the appends in front of the loading dialogue on [line 63](https://github.com/jwu910/check-it-out/pull/306/files#diff-bd9c9dcd314f2d7df52935b3a6a4d504R63)? Great catch seeing that the screen appends after the load, but I think we can make a couple more changes to just move all the appends in the right place.

Reply:
```
  screen.append(branchTable);
  screen.append(statusBarContainer);
  screen.append(loadDialogue);

  statusBarContainer.append(messageCenter);
  statusBarContainer.append(statusBar);
  statusBarContainer.append(helpDialogue);

  statusBar.append(statusBarText);
  statusBar.append(statusHelpText);
```

Could you add one additional change to this ticket? 

Looks like I duplicated `screen.append(loadDialogue)` as well on [line 118](https://github.com/jwu910/check-it-out/pull/306/files#diff-bd9c9dcd314f2d7df52935b3a6a4d504R118) Can you make a commit to just scrap this?